packages/storybook/stories/extension/_providers.tsx

```diff
@@ -1,11 +1,13 @@
 import { BootDataProvider } from '@dailydotdev/shared/src/contexts/BootProvider';
 import LogContext from '@dailydotdev/shared/src/contexts/LogContext';
-import { QueryClient, QueryClientProvider as TanStackQueryClientProvider } from '@tanstack/react-query';
-import { BootApp } from '@dailydotdev/shared/src/lib/boot';
+import {
+  QueryClient,
+  QueryClientProvider as TanStackQueryClientProvider,
+} from '@tanstack/react-query';
 import { Browser } from 'webextension-polyfill';
 import type { FC, PropsWithChildren } from 'react';
 
-const app = BootApp.Extension;
+const app = 'extension';
```

Comment:
Why did we use literal string over the enum?

Reply:
Yeah think this should just be the enum :)

Reply:
We tried to use the enum but a `Detected cycle while resolving name 'BootApp' in '/mock/boot.ts` error popped up. 

I think that fixing this issue could need more than half a day, just because we can not import the files with an absolute path and a mock for both @dailydotdev/shared/src/lib/boot' and '../lib/boot' is needed in `.storybook/main.ts` :(

We can further investigate if needed!